lib/node_modules/@stdlib/array/empty-like/test/test.js

```diff
@@ -0,0 +1,502 @@
+	];
+
+	for ( i = 0; i < values.length; i++ ) {
+		t.throws( badValue( values[i] ), TypeError, 'throws an error when provided '+values[i] );
+	}
+	t.end();
+
+	function badValue( value ) {
+		return function badValue() {
+			emptyLike( value, 'float32' );
+		};
+	}
+});
+
+tape( 'the function throws an error if provided an unrecognized data type', function test( t ) {
+	var values;
+	var i;
+
+	values = [
+		'5',
+		'beep',
+		'emptyLike',
+		'Int32',
+		'Uint32',
+		'Int16',
+		'Uint16',
+		'Int8',
+		'Uint8',
+		'Uint8c',
+		'uint8_clamped',
+		'Float64',
+		'Float32',
+		'FLOAT64',
+		'FLOAT32',
+		'GENERIC'
+	];
+
+	for ( i = 0; i < values.length; i++ ) {
+		t.throws( badValue( values[i] ), TypeError, 'throws an error when provided '+values[i] );
+	}
+	t.end();
+
+	function badValue( value ) {
+		return function badValue() {
+			emptyLike( [], value );
+		};
+	}
+});
+
+tape( 'the function returns an empty array (float64)', function test( t ) {
+	var arr;
+	var x;
+
+	x = new Float64Array( 5 );
+
+	arr = emptyLike( x );
+	t.strictEqual( instanceOf( arr, Float64Array ), true, 'returns expected value' );
+	t.strictEqual( arr.length, x.length, 'returns expected value' );
+
+	t.end();
+});
+
+tape( 'the function returns an empty array (dtype=float64)', function test( t ) {
+	var arr;
+	var x;
+
+	x = new Float32Array( 5 );
+
+	arr = emptyLike( x, 'float64' );
+	t.strictEqual( instanceOf( arr, Float64Array ), true, 'returns expected value' );
+	t.strictEqual( arr.length, x.length, 'returns expected value' );
+
+	t.end();
+});
+
+tape( 'the function returns an empty array (float32)', function test( t ) {
+	var arr;
+	var x;
+
+	x = new Float32Array( 5 );
+
+	arr = emptyLike( x );
+	t.strictEqual( instanceOf( arr, Float32Array ), true, 'returns expected value' );
+	t.strictEqual( arr.length, x.length, 'returns expected value' );
+
+	t.end();
+});
+
+tape( 'the function returns an empty array (dtype=float32)', function test( t ) {
+	var arr;
+	var x;
+
+	x = new Float64Array( 5 );
+
+	arr = emptyLike( x, 'float32' );
+	t.strictEqual( instanceOf( arr, Float32Array ), true, 'returns expected value' );
+	t.strictEqual( arr.length, x.length, 'returns expected value' );
+
+	t.end();
+});
+
+tape( 'the function returns an empty array (complex128; real)', function test( t ) {
+	var arr;
+	var x;
+
+	x = new Complex128Array( 2 );
+
+	arr = emptyLike( x );
+	t.strictEqual( instanceOf( arr, Complex128Array ), true, 'returns expected value' );
+	t.strictEqual( arr.length, x.length, 'returns expected value' );
+
+	t.end();
+});
+
+tape( 'the function returns an empty array (complex128; complex)', function test( t ) {
+	var arr;
+	var x;
+
+	x = new Complex128Array( 2 );
+
+	arr = emptyLike( x );
+	t.strictEqual( instanceOf( arr, Complex128Array ), true, 'returns expected value' );
+	t.strictEqual( arr.length, x.length, 'returns expected value' );
+
+	t.end();
+});
+
+tape( 'the function returns an empty array (dtype=complex128; real)', function test( t ) {
```

Comment:
```suggestion
tape( 'the function returns an empty array (dtype=complex128)', function test( t ) {
```

This appears to be a copy-paste bug. We don't need to distinguish between "real" and "complex". Presumably, this came from copying a file from `full-like`.